### PR TITLE
Add Krish

### LINKED ISSUE #473: Add Krish

### PR DESCRIPTION
**Pull request by krishdevdb**
Add Myself
resolve #473

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -1,4 +1,10 @@
 [
+  {
+    "name": "Krish",
+    "githubUsername": "krishdevdb",
+    "imageUrl": "https://user-images.githubusercontent.com/76587897/117103511-1db57e80-ad98-11eb-83cc-7dd9ab1f64ab.png",
+    "issueNumber": 473
+  }
```
Comment (schmelto) [inline data.json:7]:
```suggestion
  },
```
here a comma is missing if you move the coding block at the end you have to add the comma at the top :)

**Review by schmelto — CHANGES_REQUESTED** [review]
Looking good for me so far. One addition could you add yourself at the end of the `data.json` not at the top. Thanks 🚀

**Comment by krishdevdb** [comment]
Did it @schmelto

**Comment by schmelto** [comment]
resolve #473

**Comment by schmelto** [comment]
I've enhanced the PR description wit `resolve #473` thus the pr merge closes the issue.

**Review by schmelto — APPROVED** [review]

**Pull request merged**